.rubocop.yml

```diff
@@ -5,3 +5,6 @@ inherit_gem:
 inherit_mode:
   merge:
     - Exclude
+
+Style/EmptyCaseCondition:
+  Enabled: false
```

Comment:
👍 idk why rubocop thinks `elsif` is more readable 🤷

Reply:
Nor do I.

If we blindly followed Rubocop we may as well write Java.